### DISCUSSION TITLE
Enhance MessagePill

### DISCUSSION THREAD

**Discussion by mjmare** [anvilistas/anvil-extras#220] [Ideas]
Some suggestions for MessagePill:
- make the visible property accessible in the IDE
- make it optionally closable by the user
- make it close after some configurable time
- make it "stackable", ie if additional "calls" to message pill would show stacked instances of MP.

**Comment by meatballs** [comment]
Have you created this issue with a view to implementing it and submitting a pull request?

If so, they all look like good ideas to me, so please feel free to go ahead.

**Comment by mjmare** [comment]
Not really. I don't think I should inflict my programming skills on others. It was just meant as suggestions.
